src/scales/scale.time.js

```diff
@@ -36,6 +36,19 @@ module.exports = function(Chart) {
 		},
 		ticks: {
 			autoSkip: false
+		},
+		majorTicks: {
```

Comment:
If minor ticks will fetch their options from `options.ticks.minor` I think it will be breaking change or at least we'll must to refactor all other scales to use `options.ticks.minor` options.
I suggest to use your variant but without `minor` key. In that case minor ticks will use common ticks options and major ticks will use `major` options. So ticks options will looks like: 
`ticks: {
    // common tick options
    // ...
    major: {
        // overrides for major ticks only
    }
}`